lib/protocol/networks.js

```diff
@@ -671,7 +671,7 @@ testnet.deployments = {
   rollover: {
     name: 'rollover',
     bit: 1,
-    startTime: 1559347200, // June 1st, 2019
+    startTime: 1581206400, // February 9th, 2020
```

Comment:
Assuming we launch a new testnet along with main net, it would make sense to deploy these soft forks _earlier_ on testnet. That's how Bitcoin Core rolls, seems like a good idea.